### ISSUE TITLE
Add germline sex estimate to bixops file

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/ticket-tracker#523]
#### What data file(s) does this issue pertain to?
bixops file - perhaps there is a more recent one than `20230203_release_histologies_ops_update.csv` which includes these samples - if so, please send me a link to it.

#### What release are you using?

`20230203_release_histologies_ops_update.csv`

#### Put your question or report your issue here.
The following samples are missing from the ops file for germline sex estimate. Assuming paired tumor would also be missing tumor fraction/ploidy - can you please add?
[missing_germline_sex.csv](https://github.com/PediatricOpenTargets/ticket-tracker/files/10915573/missing_germline_sex.csv)

cc @zhangb1 @HuangXiaoyan0106

**Comment by HuangXiaoyan0106** [comment]
@jharenza These samples were not recorded in the `2023-02-02_histologies-base.csv`,  and gender estimation only updates the existing sample information in the base histology without adding new sample records. Therefore, there will be no gender records of these samples in the bixops file. 

The calculation of tumor fraction/ploidy is independent and does not intersect with gender prediction. As long as the tumor samples are in the histology and have genomics files, there will be calculation results.

Based on this ticket, I checked the BS ids you provided and calculated the sex results. Then, I matched the samples in histology file using the `Kids_First_Participant_ID` and updated the `germline_sex_estimate` information.
[add_germline_sex.csv](https://github.com/PediatricOpenTargets/ticket-tracker/files/10918302/add_germline_sex.csv)

@zhangb1  updated histology file:
https://cavatica.sbgenomics.com/u/d3b-bixu-ops/monthly-release-analysis/files/64083bc48de76763cec8c463/
